profiling/src/exception.rs

```diff
@@ -102,9 +106,18 @@ pub fn exception_profiling_minit() {
 }
 
 /// This initializes the `EXCEPTION_PROFILING_INTERVAL` atomic on first RINIT with the value from
-/// the INI / ENV variable. We need to initialize here even when exception profiling is turned off,
-/// as it might get enabled via a per directory setting
+/// the INI / ENV variable.
 pub fn exception_profiling_first_rinit() {
+    let exception_profiling = REQUEST_LOCALS.with(|cell| {
+        cell.try_borrow()
+            .map(|locals| locals.system_settings().profiling_exception_enabled)
+            .unwrap_or(false)
+    });
+
+    if !exception_profiling {
+        return;
+    }
+
```

Comment:
As of recently the INI is a true INI_SYSTEM, which means it can not be changed anymore (not per user dir or any other fancy things), which is also intended behaviour. With this change, we also do not show the "Exception profiling initialized .." message anymore in case it is disabled